### PR TITLE
Add guidance on rotating tokens and keys

### PR DESCRIPTION
**Pull request by poornima-krishnasamy**

### CODE REVIEW

File: source/documentation/other-topics/rotate-tokens-keys.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+## Overview
+
+Below is the guidance for rotating the tokens and keys that are set as environment variables for your application container or your automated deployment pipeline. 
+This include only the variables fetched from system secrets from a namespace and not include any application related variables.
```
Comment (davidread) [inline source/documentation/other-topics/rotate-tokens-keys.html.md.erb:13]:
```suggestion
This guide covers only secrets provided by the Cloud Platform system (your application may have other secrets too).
```

File: source/documentation/other-topics/rotate-tokens-keys.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    es-snapshot-bucket                                         Opaque                                2      144d
+    $
+    $kubectl delete secret circleci-token-XXXXX -n <yournamespace>
+
+  After the secret is deleted the replacement secret is generated immediately. Check the age of the newly generated secret to confirm. 
+  
+  Decode the secret using `cloud-platform decode-secret -n <yournamespace> -s circleci-token-XXXXX` to use it in your deployment pipeline.
+
+### AWS credentials
+   Contact `Cloud platform` team to rotate the credentials related to AWS resources such as ECR, RDS etc. The will run the instructions from the runbook [Rotate User AWS Credentials](https://runbooks.cloud-platform.service.justice.gov.uk/rotate-user-aws-credentials.html#rotate-user-aws-credentials)
```
Comment (davidread) [inline source/documentation/other-topics/rotate-tokens-keys.html.md.erb:31]:
```suggestion
   [Ask the Cloud Platform team](/documentation/reference/getting-help.html) to rotate the credentials related to AWS resources such as ECR, RDS etc., which have been [created by terraform](/documentation/deploying-an-app/add-aws-resources.html). The team will run the instructions from the runbook [Rotate User AWS Credentials](https://runbooks.cloud-platform.service.justice.gov.uk/rotate-user-aws-credentials.html#rotate-user-aws-credentials) to regenerate the AWS credentials and they will copied into the existing kubernetes secret. This means that apps that [directly access the kubernetes secret](/documentation/deploying-an-app/add-secrets-to-deployment.html) can just be restarted to obtain the fresh value.
```

**Review by davidread — COMMENTED** [review]
This is great. Just a couple of suggestions, although I'm not fully conversant in the detail in this area

File: source/documentation/other-topics/rotate-tokens-keys.html.md.erb (hunk 1 above)
Comment (poornima-krishnasamy) [reply source/documentation/other-topics/rotate-tokens-keys.html.md.erb]: Thanks @davidread . I have updated the lines

File: source/documentation/other-topics/rotate-tokens-keys.html.md.erb (hunk 2 above)
Comment (poornima-krishnasamy) [reply source/documentation/other-topics/rotate-tokens-keys.html.md.erb]: I have updated the line slightly different. The pods dont need to be restarted to get fresh values. The secrets will be updated automatically in the mountPath. Let me know your thoughts.

**Review by davidread — APPROVED** [review]
Nice one!

**Pull request merged**